### PR TITLE
mobile menu

### PR DESCRIPTION
**Pull request by Ursaaaaaa**
Added Nav Menu for mobile menu
Added Styling and CSS property
Added link to the menu section
Added JS for toggle click effects

### CODE REVIEW

File: mystyle.css [hunk 1]
```diff
@@ -523,6 +531,54 @@ textarea {
...
+  .drop-menu.active {
+    display: block;
+  }
+
+  .menu-button.active .hamburger:nth-child(2) {
+    opacity: 0;
+  }
+
+  .menu-button.active .hamburger:nth-child(1) {
+    transform: translateY(7px) rotate(45deg);
```
Comment (sja-thedude) [inline mystyle.css:550]: - [ ] Kindly make sure that mobile menu options follow the text colour as per the Figma design which is #FFF5E1.

Your design: 
Screenshot 2022-04-06 at 2 38 31 AM


Figma design:
Screenshot 2022-04-06 at 2 38 51 AM

File: index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const hamburger = document.querySelector('.menu-button');
+const menu = document.querySelector('.drop-menu');
+const logo = document.querySelector('.logo');
+const menuButton = document.querySelector('.menu-button');
+
+hamburger.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  logo.classList.toggle('active');
+  menuButton.classList.toggle('active');
+});
```
Comment (sja-thedude) [inline index.js]: - [x] Kindly make sure that when the user clicks any option of the mobile menu, the mobile menu should disappear. As you can see in the attached video that the mobile menu is persistent and doesn't disappear without clicking the cross (X) button.

https://user-images.githubusercontent.com/70269507/161868705-55d8d2e6-812b-4480-a6a1-fe78aa2d5873.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ramrod433, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Linters set up properly
✔️ Good PR Description and Title
✔️ Great job in implementing the mobile menu 👏 

## Required 
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (Ursaaaaaa) [reply index.js]: Thank you

**Review by medaminedev66 — APPROVED** [review]
## Hi @ramrod433, 

# Status: Approved ✔️
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚢
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**